src/Components/News/news.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import s from './news.module.css'
+
+const News = () => {
+    return (
+        <div className={s.main}>
+            News've been here
+    </div>
```

Comment:
I think there's a problem with indentation - don't forget to press "Shift + Alt + F".